core/imageroot/var/lib/nethserver/cluster/actions/get-cluster-status/validate-output.json

```
@@ -130,6 +144,7 @@
     "nodes",
     "leader",
     "leader_url",
-    "default_password"
+    "default_password",
+    "hostname"
```

Is `machine_id` required too?
```suggestion
    "hostname",
    "machine_id"
```